scylla/src/transport/session_test.rs

```diff
@@ -16,42 +16,66 @@ use bytes::Bytes;
 use futures::StreamExt;
 use itertools::Itertools;
 use std::collections::{BTreeMap, HashMap};
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::time::{SystemTime, UNIX_EPOCH};
 use uuid::Uuid;
 
+static UNIQUE_COUNTER: AtomicUsize = AtomicUsize::new(0);
+fn unique_name() -> String {
+    UNIQUE_COUNTER.fetch_add(1, Ordering::SeqCst);
+    format!(
```

Comment:
I'd suggest printing information about the allocated name. In case the test fails, the standard output captured during the test will be printed and the developer will learn about the keyspace name so that they can investigate its contents.